src/Psalm/Config.php

```diff
@@ -696,6 +696,7 @@ private static function loadDomDocument(string $base_dir, string $file_contents)
         $oldpwd = getcwd();
         chdir($base_dir);
 
+        assert($file_contents !== '');
```

Comment:
please document on the param instead, this way Psalm can make sure this is enforced at the source